Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/success-messages-utils.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/ban-types */
+interface IClearSuccessMessagesParams {
+    setParentSuccessMessage?: Function;
+    setSuccessMessage?: Function;
+}
+
+const clearSuccessMessages = ({
+    setParentSuccessMessage,
+    setSuccessMessage,
+} : IClearSuccessMessagesParams) => {
+    if (setSuccessMessage) {
+        setSuccessMessage();
+    }
```

Comment:
Is `clear` the correct verb here? It seems more like `set` or `reset`...

Reply:
These are the parameters for the ```clearSuccessMessages``` function, in order to make it clear that we are ```clearing``` all previous success messages, I decided to use ```clear``` as the verb. We are not setting a specific value, so I do not think ```set``` would be the one to use. When it comes to ```reset```, we should be resetting to some ```default``` value, which in the messages' case could be ```null``` or an ```empty string```.